packages/ui/src/views/repo/pull-request/compare/components/pull-request-compare-form.tsx

```diff
@@ -135,7 +131,7 @@ const PullRequestCompareForm = forwardRef<HTMLFormElement, PullRequestFormProps>
                   onDragLeave={handleDragLeave}
                   ref={dropZoneRef}
                 >
-                  <Textarea
+                  <FormInput.Textarea
                     id="description"
                     {...register('description')}
                     value={desc}
```

Comment:
Not one for this PR, but it seems weird that we're both using `register` to hook the state up to `useForm`, but also tracking and setting the values using standard state. Usually it's either/or, not both.

Reply:
Agree